### DISCUSSION TITLE
[자료공유] Git의 파일의 상태 (untracked, unmodified, modified, staged)

### DISCUSSION THREAD

**Discussion by dev-LeoKang** [4Moyede/HexatonClass-01#93] [04_자료공유]
git에서 파일의 상태를 나눌 때에는 다음과 같이 나눕니다.

* Untracked : Add 한 적 없는 파일
* Tracked : git에 한번은 등록했던 파일
    + Unmodified : Add할 것도 Commit 할 것도 없는 상태
    + Unstaged : Add가 필요한 상태
        + Modified : 파일을 수정한 상태
        + Deleted : 파일이 삭제된 상태
    + Staged  : Add 되어서 Commit 받기 준비중인 상태

한번 Tracked 되면 모든 변화에는 add, commit 이 필요하다고 생각하시면 됩니다.

**Comment by ghost** [comment]
좋은정보 감사합니다...!! 정리된 글을 봐도 실제로 예상치 못한 상황에 처하면 당황하게 된다는 ..ㅠㅠ

**Reply by jiwoo4424** [reply]
> 좋은정보 감사합니다...!! 정리된 글을 봐도 실제로 예상치 못한 상황에 처하면 당황하게 된다는 ..ㅠㅠ

공감하고 갑니다...
